error.go

```diff
@@ -6,24 +6,37 @@ import (
 	"fmt"
 )
 
-type CloudFoundryErrors struct {
```

Comment:
we drop this as its currently unused. it might have been added for handling errors of version 3 but then seems to not really have been used.